DemoApp/DemoAppCoordinator.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatUI
+import UIKit
+
+final class DemoAppCoordinator {
+    static var shared: DemoAppCoordinator!
```

Comment:
It is this ugly for two reasons:
* Initial nav controller lives in a storyboard
* LoginViewController needs that to present Chat 
* We through out LogicViewController as soon as we present Chat

Reply:
If it's ugly why don't you fix it? 🤔 🤷‍♂️

Reply:
What I meant is that we don't have much choice here, so it's ugly comparing to conventional coordinators.
Or do you mean fixing the reason I enumerated? I believe you mentioned something about storyboard being used on purpose

Reply:
Reworked it